bitnami/etcd/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: etcd
-version: 4.9.5
+version: 4.9.6
```

Comment:
Please bump minor version instead since we're adding a new parameter.

Reply:
fixed